jenkins/version-def.sh

```diff
@@ -28,7 +28,7 @@ IFS=$PRE_IFS
 
 CUDF_VER=${CUDF_VER:-"0.17"}
 CUDA_CLASSIFIER=${CUDA_CLASSIFIER:-"cuda10-1"}
-PROJECT_VER=${PROJECT_VER:-"0.3.0-SNAPSHOT"}
+PROJECT_VER=${PROJECT_VER:-"0.3.0"}
```

Comment:
This variable defines the spark-rapids jar used in integration test.
The integration tests on branch-0.3 should keep using 0.3.0-SNAPSHOT.
I don't think it should be changed here.

Reply:
correct

Reply:
I've reverted the changes to `jenkins/` files.